dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2324,15 +2323,7 @@
         },
         "script": {
           "type": "string",
-          "description": "Identifies the script of the input document.",
-          "enum": [
-            "Unknown",
-            "Latin"
```

Comment:
Is there a reason to remove this list other than it is not exhaustive?

Reply:
after the board review it was decided to keep the list and make Script an extensible enum